pydicom/tests/test_valuerep.py

```diff
@@ -185,6 +189,18 @@ def test_new_str_conversion(self):
         with pytest.raises(ValueError, match=msg):
             pydicom.valuerep.DT("a2000,00,00")
 
+    def test_str(self):
+        dt = datetime(1911, 12, 13, 21, 21, 23)
+        assert "19111213212123" == str(pydicom.valuerep.DT(dt))
+        assert "19111213212123" == str(pydicom.valuerep.DT("19111213212123"))
+        assert "1001.02.03" == str(pydicom.valuerep.DA("1001.02.03"))
```

Comment:
This isn't conformant, is it? Oh, it's there for backwards compatibility to ACR-NEMA 300. 

Maybe we should just update the old example files the source talks about and/or force '.' separated values to be conformant.

Reply:
#1366